src/commands/build.rs

```diff
@@ -38,7 +38,7 @@ pub struct BuildCommand {
 impl BuildCommand {
     pub async fn run(self) -> Result<()> {
         let manifest_file = crate::manifest::resolve_file_path(&self.app_source)?;
-        spin_build::build(&manifest_file).await?;
+        spin_build::build(&manifest_file, &self.component_id).await?;
```

Comment:
Would it make sense for `component_id` to be a `Vec<String>` (allowing the flag to be specified multiple times)? I'm thinking that would give feature parity with the zombie PR https://github.com/fermyon/spin/pull/903

Reply:
It couldn't possibly do any harm, he said, cheerily placing his Jenga block.